src/style.css

```diff
@@ -1,3 +1,72 @@
+html {
+  font-family: Roboto, Arial, sans-serif;
+  font-size: 12px;
+  font-weight: 500;
+  line-height: 14.06px;
```

Comment:
The `line-height` value is quite specific and unusual as a pixel value. Usually, `line-height` is set without units or as a relative value to the font-size, such as `line-height: 1.2;` for better scalability and readability across different devices.